### ISSUE TITLE
Transport information Email empty

### ISSUE THREAD

**Issue by ursulafeu** [viszerale-therapie/simple-courses#207]
Beim Kurs mit der ID 384(Anatomie intraperitoneal & Thorax) habe ich gestern die Transport information ausgeschickt. Die Email kam leer bei den TN an.

**Comment by ursulafeu** [comment]
siehe auch Ticket #185

**Comment by Andreas-Schoenefeldt** [comment]
Ach Herrje, das schaue ich mir ASAP an (durch die Tour und so kann es etwas durcheinander kommen, sollte aber irgendwie reinhüpfen).

**Comment by ursulafeu** [comment]
lieber Andreas, wie schauts damit aus? Die Emails sollen raus ... danke!

**Comment by Andreas-Schoenefeldt** [comment]
Hi Ursula,

ich habe eine kleinigkeit gefunden, aber sonst sieht alles gut aus. Knnst du es nochmal probieren?

**Comment by ursulafeu** [comment]
ich denke es funktioniert noch nicht. Ich habe es grade nochmal getestet beim Kurs 378 – da ist Florinda Vortragende und da kommt normalerweise die Transport Email cc an mich (weil da die info-Email hinterlegt ist). Da kommt aber keine Email...

**Comment by Andreas-Schoenefeldt** [comment]
Die mails gehen raus, aber nicht an den Lehrer - ich prüfe das nochmal.

**Comment by Andreas-Schoenefeldt** [comment]
Hallo Urusla,

also laut https://github.com/viszerale-therapie/simple-courses/issues/156 haben wir die kopie an den Lehrer und Assitenten absichtlich aus DSGVO Gründen ausgebaut. 

Wenn es wieder rein soll, bitte eine kurze Rückmeldung.

1000 Dank :)
Andreas

**Comment by ursulafeu** [comment]
nein, das passt so. Aber vielleicht kannst du die office Email generell auf cc setzen, damit ich sehe, ob es raus ging?
